docs/DatabaseCatalogAPI.yaml

```diff
@@ -196,6 +196,24 @@ paths:
               schema:
                 $ref: "#/components/schemas/GtfsDatasets"
 
+  /v1/gtfs_feeds/{id}/gtfs_rt_feeds:
+    parameters:
+      - $ref: "#/components/parameters/feed_id_of_datasets_path_param"
```

Comment:
I'm being picky, but the description of `feed_id_of_datasets_path_param` says: `The ID of the feed for which to obtain datasets` which is not quite right.
Maybe you could use [feed_id_path_param](https://github.com/MobilityData/mobility-feed-api/blame/5b3c60410bae78475d95982c7e023d5789eb456e/docs/DatabaseCatalogAPI.yaml#L928C5-L928C23) instead?